image.go

```diff
@@ -813,3 +813,7 @@ func (b *Builder) makeImageRef(options CommitOptions) (types.ImageReference, err
 	}
 	return ref, nil
 }
+
+func (b *Builder) MakeImageRef(options CommitOptions) (types.ImageReference, error) {
```

Comment:
This method doesn't need to exist.  Just have `Commit()` call `MakeContainerImageRef()` directly.

Reply:
`Commit(` is expecting type `types.ImageReference` but `MakeContainerImageRef` returns `*ContainerImageRef`, function is just helping me avoid recasting in the code but I'm can change this if there is better casting method.

Reply:
this is fixed.